File: src/main/java/com/koddy/server/coffeechat/application/usecase/query/GetMenteeCoffeeChats.java
package com.koddy.server.coffeechat.application.usecase.query;

import com.koddy.server.coffeechat.domain.model.CoffeeChatStatus;

import java.util.List;

public record GetMenteeCoffeeChats(
        long menteeId,
        List<CoffeeChatStatus> status,
        int page
) {
}

File: src/main/java/com/koddy/server/coffeechat/application/usecase/query/GetMenteeCoffeeChats.kt
package com.koddy.server.coffeechat.application.usecase.query

import com.koddy.server.coffeechat.domain.model.CoffeeChatStatus

data class GetMenteeCoffeeChats(
    val menteeId: Long,
    val status: List<CoffeeChatStatus>,
    val page: Int,
)
